### DISCUSSION TITLE
Test specific release - v8.0.7 for example, wich branch I have to use?

### DISCUSSION THREAD

**Discussion by danielancines** [dotnet/maui#20950] [General]
I need to test some behavior against version v8.0.7 (latest release), wich branch I must use on Maui repo?
I think the main branch could be not the exactly de same as the released version.

Any ideias?

**Comment by Pastajello** [comment]
I recommend using tags for this:
<img width="373" alt="image" src="https://github.com/dotnet/maui/assets/28345098/c73a1f73-08d2-4975-b0ef-0f6bc840df60">

**Comment by danielancines** [comment]
Thank you @Pastajello , I did earlier I opened this discussion.
